### ISSUE TITLE
windows bug when compiling a single file

### ISSUE THREAD

**Issue by dissolve** [dlmanning/gulp-sass#756]
So this is very strange behaviour that i'm not able to trackdown exactly what is causing it

```
var sass = require('gulp-sass');

gulp.src(srcFiles, {sourcemaps = '.', newer: lastRun(sometask)})
    .pipe(sass())
    .pipe(gulp.dest('./dist'));
```

any time i change a file, it works perfectly.
any time i change multiple files, it works perfectly.
any time i update a modified at on multiple files (by like a touch *.scss command) it works perfectly
if I update a modified timestamp on a SINGLE file (on Windows only) it will create a .css file in the source directory.

in other words
if I have  src/sass/base.scss it will create src/sass/base.css .
It will still create the correct file in the /dist directory, but it for some reason also dumps one out in the source file

I have no idea where the bug is, so i'm starting here in hopes someone knows what is going on, and if its not here, where I need to file a bug
